conda/base/context.py

```diff
@@ -314,6 +314,7 @@ class Context(Configuration):
     ignore_pinned = ParameterLoader(PrimitiveParameter(False))
     report_errors = ParameterLoader(PrimitiveParameter(None, element_type=(bool, NoneType)))
     shortcuts = ParameterLoader(PrimitiveParameter(True))
+    disable_channel_notices = ParameterLoader(PrimitiveParameter(False))
```

Comment:
I'm a bit skeptical of parameter overloading like this. How about we just have a separate "number_channel_notices" parameter that can be set to any integer value between 1 and 100? (defaulting to 5)

Reply:
Skeptical as a general rule of thumb or specifically to our implementation?

Reply:
I'm skeptical of this kind of thing in general. I wouldn't want to be able to assign an integer configuration to a boolean value, and then have it resolve to `3`. This could be confusing to users.

How about we just have the `number_channel_notices` parameter defined and tell users they can set it to `0` to completely disable them?